src/test/span_tests.cpp

```diff
@@ -53,7 +53,7 @@ BOOST_AUTO_TEST_SUITE(span_tests)
 // aren't compatible with Spans at compile time.
 //
 // Previously there was a bug where writing a SFINAE check for vector<bool> was
-// not possible, because in libstdc++ vector<bool> has a data() memeber
+// not possible, because in libstdc++ vector<bool> has a data() member
```

Comment:
completely unrelated, but got this warning from the CI

Reply:
Hmm not sure we should have these changes in an unrelated PR. The CI failure was for a whitespace issue in the hex parsing code.

Reply:
Moved it to a separate PR https://github.com/bitcoin/bitcoin/pull/29471